src/rendering/RenderUtil.cc

```diff
@@ -1281,6 +1282,15 @@ void RenderUtil::ShowGrid()
   root->AddChild(visual);
 }
 
+/////////////////////////////////////////////////
+void RenderUtil::ShowOriginAxes()
+{
+  rendering::VisualPtr root = this->dataPtr->scene->RootVisual();
+
+  auto worldOriginAxes = this->dataPtr->scene->CreateAxisVisual("worldOriginAxes");
```

Comment:
Now that the plugin displays entity axes. I think words like "origin" and "world" can be misleading.

I'd actually suggest removing this logic from `RenderUtil` and creating the axes directly from `AxesConfig`, within the rendering thread.